### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60734]
#### VFS team name
Digitize Forms

#### VFS product name
Form 21-10210 Lay/Witness Statement

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
When filing to support someone else, on Step 5 of 10 (/statement-c) and Step 6 of 10 (/veteran-personal-information), there's text that's styled like a `legend` but is coded as just a `span` ("Tell us about the claimed issue that you’re addressing" and "Tell us about the Veteran you’re connected to"). For screen reader users who tab into the first input on the page, that text won't be announced because it's not programmatically associated with the input.

The practical impact on Step 5 isn't that big, since the `label` is sufficient to communicate what's needed for this step. But on Step 6, that's useful context to help clarify who's information is being submitted. It's not strictly invalid HTML, since the `legend` is technically an optional child of a `fieldset`. But it's a little unusual and isn't consistent with other forms on VA.gov are structured.

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

### Platform Recommendation
Make this text a `legend` unless there's a reason it shouldn't be.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jeana-adhoc** [comment]
Hi @tlei123 -

The issue listed here is similar to the ones we've been working on in 21-0845.

The following pages have items that should be a `<legend>` but are currently `<span>`s

/supporting-forms-for-claims/lay-witness-statement-form-21-10210/statement-a
`Tell us about the claimed issue that you’re addressing on behalf of the Veteran`

/supporting-forms-for-claims/lay-witness-statement-form-21-10210/statement-b
`Tell us about the claimed issue that you’re addressing`


/supporting-forms-for-claims/lay-witness-statement-form-21-10210/statement-c
`Tell us about the claimed issue that you’re addressing`

/supporting-forms-for-claims/lay-witness-statement-form-21-10210/statement-d
`Provide your statement`

/supporting-forms-for-claims/lay-witness-statement-form-21-10210/claimant-personal-information
`Tell us about the person who has the existing claim`

/supporting-forms-for-claims/lay-witness-statement-form-21-10210/veteran-personal-information
`Tell us about the Veteran you’re connected to`

**Comment by tlei123** [comment]
PR opened.  Should be approved & merged by tmrw (Tue) ~Noon ET.

**Comment by tlei123** [comment]
@jeana-adhoc , PR now merged.  Should be in Staging by ~ 3:30 E.

**Comment by jeana-adhoc** [comment]
This looks good! Thanks, @tlei123 !
